static/js/components/PersonalForm.js

```diff
@@ -152,6 +152,10 @@ export default class PersonalForm extends ProfileFormFields {
             </Cell>
           </Grid>
         </section>
+        <section>
+          <h3>Telephone</h3>
+          { this.boundTelephoneInput(['phone_number']) }
```

Comment:
This telephone input isn't accessible. The `h3` tag above should be marked as a label for this input.